security/container_content.adoc

```diff
@@ -0,0 +1,64 @@
+[[security-content]]
+= Container Content
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-content-whats-in-your-container]]
+== What's in Your Container?
+
+When it comes to security, what is inside your container matters. Applications
+and infrastructures are composed of readily available components, many of which
+are open source packages such as, the Linux operating system, JBoss Web Server,
+PostgreSQL, and Node.js.
+
+Containerized versions of these packages are now also readily available, so that
+you do not have to build your own. But, as with any code you download from some
+external source, you need to know where they originally came from, who built
+them, and whether there is any malicious code inside them. Ask yourself:
+
+- Will what is inside the containers compromise your infrastructure?
+- Are there known vulnerabilities in the application layer?
+- Are the runtime and OS layers up to date?
+
+[discrete]
+[[security-content-further-reading-1]]
+==== Further Reading
+
+- xref:{sourcedir}using_images/index.adoc#using-images-index[_{product-title} Using Images_]
+** Reference documentation on framework, database, and service container images
+provided by Red Hat for use on {product-title}
+
```

Comment:
Should we also link to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/container_security_guide/